plugin.js

```diff
@@ -61,8 +63,21 @@ function onReqHandlerWrapper (fastify) {
   }
 }
 
+function getHook (hook, next) {
+  const hooks = {
+    onRequest: 'onRequest',
+    preParsing: 'preParsing',
+    preValidation: 'preValidation',
+    preHandler: 'preHandler',
+    [false]: false
+  }
+
+  return hooks[hook] ?? next(new Error('You can set false or use only \'onRequest\' , \'preParsing\' , \'preValidation\' , \'preHandler\' hooks'))
```

Comment:
I think rephrasing would help. prefixing it with @fastify/cookie helps to debug it faster. 

```suggestion
  return hooks[hook] ?? next(new Error('@fastify/cookie: Invalid value provided for the hook-option. You can set the hook-option only to false, \'onRequest\' , \'preParsing\' , \'preValidation\' or \'preHandler\''))
```